plugins/last-fm.py

```diff
@@ -0,0 +1,237 @@
+# last.fm plugin
+#
+# usage:
+# .np [lastfm_nickname] --- check what is playing right now for an user
+# NOTE: this bot saves the last lastfm nick you used. You *really* only
+# need to use this command like that once, afterwards you can just do
+# .np , and the bot will remember the lastfm nick associated with your
+# irc nick.
+# .np @[irc_username] --- check what's playing for an another user (if
+# they have their nicks saved)
+#
+#
+# TODO:
+# .compat [lastfm_nickname] [lastfm_nickname] --- get compatiblity
+# between two lastfm users.
+# NOTE .compat can be used like this too:
+# .compat @[irc_nick] (compares your saved nick with the argument nick)
+# .compat [lastfm_nick] (same as above, but with a lastfm nick)
+# .compat @[irc_nick] [lastfm_nick]
+# .compat [irc_nick] @[lastfm_nick]
+# .compat @[irc_nick] @[irc_nick]
+
+
+from typing import List
+import asyncio
+import requests
+from datetime import datetime
+
+# import core db functions
+from core import db, hook
+
+# db table definition
+quote_columns: List[str] = [
+    'irc_nick', 'lastfm_nick']
+
+api_url = 'http://ws.audioscrobbler.com/2.0/?format=json'
+
+
+def _get_lastfm_nick(conn, irc_nick):
+    nicks = db.get_row(conn, 'lastfm', 'irc_nick', irc_nick)
+    try:
+        return nicks[0][1]
+    except IndexError:
+        pass
```

Comment:
Could just return None here, that then should remove the need for `nick_check = None` on line 48. Also line 51 you could probably do `if not nick_check:`.